R/generate_initial_ranking.R

```diff
@@ -13,6 +13,9 @@
 #' @param cl Optional computing cluster used for parallelization, returned
 #' from \code{parallel::makeCluster}. Defaults to \code{NULL}.
 #'
+#' @param random Logical specifying whether or not to use a random initial ranking.
+#'   Defaults to \code{FALSE}.
+#'
```

Comment:
Looks like a nice addition! How about adding a @note mentioning the caveat you posted (about the possible combinatoric explosion when there are many degrees of freedom)?

Reply:
Very good point! I'll do :)
Will also try today and see if I can improve it somewhat